### ISSUE TITLE
setup script should open required ports in firewall

### ISSUE THREAD

**Issue by Ottovsky** [tinkerbell/tink#113]
Hey,

Currently the setup script assumes that all the required ports on the target hosts are open in firewall. It is not always a case, for example centos 7.6 comes with a default rule to drop all traffic which does not match firewall rules. It would be good to open the required ports: 67, 69, 443 during the script execution, so tinkerbell works out of the box.

**Comment by alexellis** [comment]
Is this related to #90?

**Comment by Ottovsky** [comment]
I am not entirely sure about this, my use case was focused on the on-premise environment without direct exposure to the internet.

**Comment by gianarb** [comment]
I will close this for now. The setup script intended for the user is now moved to https://github.com/tinkerbell/sandbox. 
Anyway, I don't think a setup bash script can cover all the operation experience required to have a system up and running. 

It is written to boost a possible environment (a simple one) that can be used as the first test.

We do not have yet a story around ops excellence for Tinkerbell, but I doubt it will use the `setup.sh` we provide as getting started on Packet and Vagrant. It is also hard to imagine a solution that works for everyone.

we started to document each service independently and at some point, it will cover the required port and their purpose as well.

I am closing this because I doubt the setup.sh will ever deal with iptables or similar
